code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -12,6 +12,52 @@
 	use_sound = 'sound/items/storage/briefcase.ogg'
 	drop_sound = 'sound/items/drop/backpack.ogg'
 
+	var/open = 0
+
+	examine(mob/user)
+		if(..(user, 1))
+			user << text("The breifcase is [src.open ? "open" : "closed"].")
+
+	verb/open_case()
+		set name = "Open Briefcase"
+		set category = "Object"
+		set src in view(1)
+
+		if(usr.stat || usr.restrained() || usr.lying || !istype(usr, /mob/living))
+			to_chat(usr, "<span class='warning'>You can't do that.</span>")
+			return open
+
+		if(!Adjacent(usr))
+			to_chat(usr, "You can't reach it.")
+			return open
+
+		if(!istype(loc,/turf))
+			to_chat(usr, "[src] is too bulky!  You'll have to set it down.")
+			return open
+
+		if(open == 1)
+			icon_state = "briefcase"
+			return 0
+		else
+			icon_state = "briefcaseopen"
+			return 1
+
+	AltClick()
+		open = open_case()

```

Comment:
This *definitely* doesn't need to be a verb since you just use altclick anyways.